pages/leaderboard.tsx

```diff
@@ -117,6 +114,7 @@ export default function Leaderboard({ loginContext }: Props) {
 
   const { checkLoggedIn, checkLoading } = loginContext
   const isLoggedIn = checkLoggedIn()
+  const ended = phase === 1
```

Comment:
This logically is a little confusing to me -- I'd expect ended to be true if the phase isn't 1. Not a big deal though, it works